### PR TITLE
refactor and looping

### PR DESCRIPTION
**Pull request by reidbaker**
UI might need some polish from @ejohnson44 but it works

### CODE REVIEW

**Comment by dgreenhalgh** [comment]
I like it. Who wrote this code before it was refactored? It may be useful to have that person take a look at it.

**Comment by reidbaker** [comment]
Me :)

**Comment by dgreenhalgh** [comment]
Hahaha

File: fanClub/src/com/lastcrusade/fanclub/model/Playlist.java [hunk 1]
```diff
@@ -7,92 +7,52 @@
...
         return removeMeta;
     }
 
-    public List<SongMetadata> getUnPlayedList() {
-        return unPlayed;
-    }
-    
-    public List<SongMetadata> getPlayedList(){
-        return played;
+    public List<SongMetadata> getSongsToPlay(){
```
Comment (bryansills) [inline fanClub/src/com/lastcrusade/fanclub/model/Playlist.java:35]: is this the best name for this method? i feel like with this name we should only get the unplayed songs.

**Pull request merged**